src/constants/tokenLists/mainnet.tokenlist.ts

```diff
@@ -128,23 +128,23 @@ export const MAINNET_TOKEN_LIST = {
       symbol: 'jEUR',
       name: 'Jarvis Synthetic Euro',
       decimals: 18,
-      logoURI: 'https://i.imgur.com/ZvP634G.png'
+      logoURI: 'https://imgur.com/a/GnVghwz.png'
```

Comment:
please use the direct link (you can right-click at the image and select "Copy image address")
The direct link is something like "https://i.imgur.com/...."

Similar to other links

Reply:
it's still not a direct link, so it does not work

Please close this issue, I will handle this update for you